java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -540,4 +546,48 @@ void testEmptyStringCatColumnOpts() {
       }
     }
   }
-}
\ No newline at end of file
+
+  @Test
+  void testNVStringManipulationWithNulls() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    try (ColumnVector v = ColumnVector.fromStrings("a", "B", "cd", "\u0480\u0481", "E\tf",
+                                                   "g\nH", "IJ\"\u0100\u0101\u0500\u0501",
+                                                   "kl m", "Nop1", "\\qRs2", null,
+                                                   "3tuV\'", "wX4Yz", "\ud720\ud721");
+         ColumnVector e_lower = ColumnVector.fromStrings("a", "b", "cd", "\u0481\u0481", "e\tf",
+                                                         "g\nh", "ij\"\u0101\u0101\u0501\u0501",
+                                                         "kl m", "nop1", "\\qrs2", null,
+                                                         "3tuv\'", "wx4yz", "\ud720\ud721");
+         ColumnVector e_upper = ColumnVector.fromStrings("A", "B", "CD", "\u0480\u0480", "E\tF",
+                                                         "G\nH", "IJ\"\u0100\u0100\u0500\u0500",
+                                                         "KL M", "NOP1", "\\QRS2", null,
+                                                         "3TUV\'", "WX4YZ", "\ud720\ud721");
+         ColumnVector lower = v.lower();
+         ColumnVector upper = v.upper()) {
```

Comment:
`\ud720\ud721` is the same in lowercase or uppercase?

Reply:
could you add a comment just documenting what special characters you are using in the test?